src/pages/AdminDashboard/roomsReducer.ts

```diff
@@ -0,0 +1,18 @@
+import produce, { Draft } from 'immer'
+import { SET_ROOMS } from './roomsActionTypes'
+import { RoomsState } from './roomsSelector'
+
+const initialState = {
+    rooms: [],
+}
+
+export const roomsReducer = produce(
+    (draft: Draft<RoomsState>, { type, payload }) => {
+        switch (type) {
+            case SET_ROOMS:
```

Comment:
add another case on lagout to return the initialState

Reply:
is here the best spot ? On logout case I guess we may need to reset several parts of the state. I'd not like to add a reset action in multiple spots

Reply:
well, I'm not sure we can reset eveyrthing in every state. Every reducer manage their own reset/logout so we can have a fine details about what's going on.

Reply:
yes but for an operation like logout, where we have multiple state parts to reset, isn't there a way to do it in a single spot instead of polluting multiple reducers with reset logic ? Like in the `rootReducer` ?